### PR TITLE
[CAS-1173] Add Hindi language support

### PR DESCRIPTION
**Pull request by amitkma**
https://stream-io.atlassian.net/browse/CAS-1173

### 🎯 Goal

Add Hindi translations 

### 🛠 Implementation details
Added hindi translations for all corresponding english resources.

You can find English strings here:
- [Client](https://github.com/GetStream/stream-chat-android/blob/develop/stream-chat-android-client/src/main/res/values/strings.xml)
- [Common](https://github.com/GetStream/stream-chat-android/blob/develop/stream-chat-android-ui-common/src/main/res/values/strings.xml)
- [UI Components](https://github.com/GetStream/stream-chat-android/tree/develop/stream-chat-android-ui-components/src/main/res/values)

### ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (docusaurus, tutorial, CMS (task created))
- [x] Reviewers added

### CODE REVIEW

File: stream-chat-android-ui-components/src/main/res/values-hi/strings_common.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <string name="stream_ui_mention">\@%s</string>
+    <string name="stream_ui_yesterday">कल</string>
+
+    <!-- User Status (Used in Channel Actions Dialog and Message List Header) -->
+    <string name="stream_ui_user_status_online">ऑनलाइन</string>
+    <string name="stream_ui_user_status_last_seen">पिछली बार देखा %s</string>
+    <string name="stream_ui_user_status_last_seen_just_now">अंतिम बार अभी देखा</string>
+
+    <!-- Message Preview (User in Mention List and Search Result List) -->
+    <string name="stream_ui_message_preview_file">File: %s</string>
```
Comment (adasiewiczr) [inline stream-chat-android-ui-components/src/main/res/values-hi/strings_common.xml:12]: Did you intentionally leave this in English?

File: stream-chat-android-ui-components/src/main/res/values-hi/strings_mention_list.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="stream_ui_mention_list_empty">No mentions exist yet…</string>
```
Comment (adasiewiczr) [inline stream-chat-android-ui-components/src/main/res/values-hi/strings_mention_list.xml:3]: Same here

File: stream-chat-android-ui-components/src/main/res/values-hi/strings_common.xml (hunk 1 above)
Comment (amitkma) [reply stream-chat-android-ui-components/src/main/res/values-hi/strings_common.xml:12]: Yes, I think the file attachments name would be in english so this string will have hindi + english combined in one statement. What do you suggest?

File: stream-chat-android-ui-components/src/main/res/values-hi/strings_mention_list.xml (hunk 2 above)
Comment (amitkma) [reply stream-chat-android-ui-components/src/main/res/values-hi/strings_mention_list.xml]: Oops! Done now.

**Review by adasiewiczr — APPROVED** [review]

**Pull request merged**